simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/websocket/WebSocketMessageInbox.java

```diff
@@ -0,0 +1,41 @@
+package com.evbox.everon.ocpp.simulator.websocket;
+
+import com.evbox.everon.ocpp.simulator.station.exceptions.StationException;
+
+import java.util.Comparator;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.PriorityBlockingQueue;
+
+public class WebSocketMessageInbox {
+
+    private final BlockingQueue<WebSocketClientInboxMessage> webSocketMessageInbox;
+
+    public WebSocketMessageInbox() {
+        this.webSocketMessageInbox = new PriorityBlockingQueue<>(1,
+                Comparator.comparing(WebSocketClientInboxMessage::getPriority).thenComparing(WebSocketClientInboxMessage::getSequenceId));
+    }
+
+    /**
+     * Inserts the message into this inbox. If no space available throws {@link StationException}.
```

Comment:
Either use @throws in both comments or in neither of them.

Reply:
done